src/communication/bolt/v1/codes.hpp

```diff
@@ -16,7 +16,8 @@
 namespace memgraph::communication::bolt {
 
 inline constexpr uint8_t kPreamble[4] = {0x60, 0x60, 0xB0, 0x17};
-inline constexpr uint8_t kProtocol[4] = {0x00, 0x00, 0x00, 0x01};
+inline constexpr uint8_t kProtocol[4] = {0x00, 0x00, 0x03, 0x04};
```

Comment:
I think it is confusing to see `kProtocol` being defined here since it has nothing to do with the server but only the client. Maybe it makes sense to move this to the client code, plus it is only used in the client code